### DISCUSSION TITLE
Collection of minor localization-related problems

### DISCUSSION THREAD

**Discussion by User670** [26F-Studio/Techmino#538] [General]
I don't feel like creating issues for them separately

also, 1. I'll not put missing translations here, only issues with existing translations (wrong/questionable translations, text too long/unsupported, etc), and 2. I'll delete fixed issues from here.

- French: also, is the colon used like this?
![image](https://user-images.githubusercontent.com/22617255/144983487-841df1ee-22b7-4eae-ae38-5fd38cb39842.png)
- Traditional Chinese. Regarding "Line" (as in the thing you clear), current traditional Chinese localization retained the Mainland name of 行. In Tetris Effect, lines are referred to as 层 in Traditional Chinese; while in Puyo Puyo Tetris 2, whenever it mentions "clearing lines", it always used the English word "LINE", and sometimes refers to "a row of blocks" as 列. Using 行 might be not exactly correct in Taiwan, for 行 and 列 have oppposite meanings in Mainland and in Taiwan. Maybe consider using 层 to refer to lines instead?

**Comment by MrZ626** [comment]
~~close #538~~
